packages/vuetify/src/components/VProgressCircular/_variables.scss

```diff
@@ -3,5 +3,4 @@
 $progress-circular-rotate-animation: progress-circular-rotate 1.4s linear infinite !default;
 $progress-circular-rotate-dash: progress-circular-dash 1.4s ease-in-out infinite !default;
 $process-circular-intermediate-svg-transition: all .2s ease-in-out !default;
-$progress-circular-underlay-stroke: rgba(map-get($shades, 'black'), 0.1) !default;
```

Comment:
This is a breaking change.

Reply:
Went back and realized it was only working on Firefox. Decided to switch out black for grey instead. Attached screenshots of how it looks now.

<img width="88" alt="Screenshot 2020-10-05 at 18 09 26" src="https://user-images.githubusercontent.com/13445576/95086839-925f0680-073e-11eb-83df-b0a47765554d.png">
<img width="81" alt="Screenshot 2020-10-05 at 18 09 34" src="https://user-images.githubusercontent.com/13445576/95086844-9428ca00-073e-11eb-8c59-6ae973da2fa8.png">

Reply:
@johnleider Screenshots above of the progressbar with grey as underlay color